pkg/loop/internal/pb/reporting.proto

```diff
@@ -0,0 +1,122 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-relay/pkg/loop";
+
+package loop;
+
+import "google/protobuf/empty.proto";
+import "relayer.proto";
+
+service ReportingPluginFactory {
+  rpc NewReportingPlugin (NewReportingPluginRequest) returns (NewReportingPluginReply) {}
+}
+
+message NewReportingPluginRequest {
+  ReportingPluginConfig reportingPluginConfig = 1;
+}
+
+message NewReportingPluginReply {
+  uint32 reportingPluginID = 1;
+  ReportingPluginInfo reportingPluginInfo = 2;
+}
+
+// ReportingPluginConfig represents [github.com/smartcontractkit/libocr/offchainreporting2/types.ReportingPluginConfig]
+message ReportingPluginConfig {
+  bytes configDigest = 1; // [32]byte
+  uint32 oracleID = 2; // uint8
+  uint32 n = 3;
+  uint32 f = 4;
+  bytes onchainConfig = 5;
+  bytes offchainConfig = 6;
+  int64 estimatedRoundInterval = 7;
```

Comment:
Could we include interval unit in the name?
estimatedRoundIntervalSeconds

Reply:
Hmmm I would like to keep the names aligned with the Go type if possible. Is it more clear if we include that a note that these are just `time.Duration`s?
```suggestion
  int64 estimatedRoundInterval = 7; // time.Duration
```